### ISSUE TITLE
OS X: "Can't assign requested address"

### ISSUE THREAD

**Issue by charlesfries** [OpenBazaar/OpenBazaar#1335]
Having some IP problems. Running OS X 10.10.3

**Comment by hoffmabc** [comment]
Did you run ./configure.sh and then ./openbazaar start

**Comment by charlesfries** [comment]
Yes I did. I am using my University network, would that be the source of the problem?

**Comment by hoffmabc** [comment]
Could be! I could not get it working on super restrictive networks. Can you copy paste your logs/production.log file?

**Comment by hoffmabc** [comment]
Run:

```
./openbazaar --log-level 5 start 
```
and post the log for us.

**Comment by charlesfries** [comment]
Is this what you are looking for? http://pastebin.com/HK2xBadP

**Comment by hoffmabc** [comment]
This looks like you are using an old version of the software.

**Comment by charlesfries** [comment]
Literally just recloned before I got that log.

**Comment by hoffmabc** [comment]
Hmm don't usually hit the peers table. Is it on log level 5?

**Comment by charlesfries** [comment]
Yes it is.

**Comment by charlesfries** [comment]
Recloned once more, working now. However I still cannot add market GUIDs. Same system as above.
